source/standards/cts1.rst

```diff
@@ -0,0 +1,673 @@
+  The token ID could be an encoding of a small text string or some checksum hash, but to save energy it is still recommended to use small token IDs if possible.
+
+.. _CTS-TokenAmount:
+
+``TokenAmount``
+^^^^^^^^^^^^^^^
+
+An amount of a token type is an unsigned 64 bit integer.
+
+It is serialized using 8 bytes little endian::
+
+  TokenAmount ::= (amount: Byte⁸)
+
+.. _CTS-ReceiveHookName:
+
+``ReceiveHookName``
+^^^^^^^^^^^^^^^^^^^
+
+A smart contract receive function name.
+A receive function name is prefixed with the contract name, followed by a ``.`` and a name for the function.
+It MUST consist only of ASCII alphanumeric or punctuation characters.
+The contract name is not allowed to contain ``.``.
+
+It is serialized as: the function name byte length (``n``) is represented by the first 2 bytes, followed this many bytes for the function name (``name``).
+The receive function name MUST be 100 bytes or less::
+
+  ReceiveHookName ::= (n: Byte²) (name: Byteⁿ)
+
+.. note::
+
+  This type is passed in a parameter for smart contract function calls, be aware of the parameter size limit of 1024 bytes.
+
+.. _CTS-ContractName:
+
+``ContractName``
+^^^^^^^^^^^^^^^^
+
+A name of a smart contract.
+It must be prefixed with ``init_`` and MUST consist only of ASCII alphanumeric or punctuation characters.
+The contract name is not allowed to contain ``.``.
+
+It is serialized as: the contract name byte length (``n``) is represented by the first 2 bytes, followed this many bytes for the contract name (``name``).
+The contract name MUST be 100 bytes or less::
+
+  ContractName ::= (n: Byte²) (name: Byteⁿ)
+
+.. _CTS-AccountAddress:
+
+``AccountAddress``
+^^^^^^^^^^^^^^^^^^^
+
+An address of an account.
+
+It is serialized as 32 bytes::
+
+  AccountAddress ::= (address: Byte³²)
+
+.. _CTS-ContractAddress:
+
+``ContractAddress``
+^^^^^^^^^^^^^^^^^^^
+
+An address of a contract instance.
+It consists of an index and a subindex both unsigned 64 bit integers.
+
+It is serialized as: first 8 bytes for the index (``index``) followed by 8 bytes for the subindex (``subindex``) both little endian::
+
+  ContractAddress ::= (index: Byte⁸) (subindex: Byte⁸)
+
+.. _CTS-Address:
+
+``Address``
+^^^^^^^^^^^
+
+Is either an account address or a contract address.
+
+It is serialized as: First byte indicates whether it is an account address or a contract address.
+In case the first byte is 0 then ``AccountAddress`` (``address``) is followed.
+In case the first byte is 1 then ``ContractAddress`` (``address``) is followed::
+
+  Address ::= (0: Byte) (address: AccountAddress)
+            | (1: Byte) (address: ContractAddress)
+
+
+.. _CTS-Receiver:
+
+``Receiver``
+^^^^^^^^^^^^
+
+The receiving address of a transfer, which is either an account address or a contract address.
+In the case of a contract address, additional information such as the name of a receive function and some additional bytes to call on the receiving contract.
```

Comment:
No this is actually not relevant anymore, since the additional data was at some point only for when contracts received tokens.
Now every transfer can include additional data.

I'll rephrase this and remove `additional bytes` from here